test/controllers/PrepareUploadControllerSpec.scala

```diff
@@ -24,39 +24,54 @@ class PrepareUploadControllerSpec extends UnitSpec with Matchers {
 
       "build and return upload URL if valid request" in {
 
+        //given
```

Comment:
There is a given when then style of tests in scalatest which could be used as well